docs/introduction/hosting-and-publishing.md

```diff
@@ -0,0 +1,223 @@
+  panel to upload assets and get CDN URLs in return.
+- [imgur](https://imgur.com/) - For images, we could use imgur, a popular image
+  hosting service.
+
+### Hosting Models
+
+Hosting models is not as simple. Models usually come as groups of files in a
+folder, where the model file relatively references other files such as images.
+Thus, models have to be uploaded as a single folder in the same directory. Many
+of the free asset hosting services support only uploading one file a time. One
+solution would be to rename all of the image paths to the CDN paths after the
+images are individually uploaded, but that is tedious. There are a couple known
+solutions for easily hosting models via CDN:
+
+If publishing a site via Neocities, we can upload any number of files and
+directories into the site directory:
+
+![Neocities](https://cloud.githubusercontent.com/assets/674727/25639880/713c8266-2f42-11e7-9f2a-8e552bda80fa.jpg)
+
+*Neocities Asset Uploader*
+
+Or we can upload assets to a GitHub repository, and use GitHub to serve the
+model files.
+
+1. Go to one of our GitHub repositories.
+2. Click **Upload files**.
+3. Upload our assets and wait for the upload to finish.
+4. Type a quick message at the bottom and hit **Commit changes**.
+5. Wait for processing.
+6. Once finished, click on the primary asset file.
+7. Click on **Raw**.
+8. Then we have our asset URL hosted on GitHub. We can also change
+`raw.githubusercontent.com` to `cdn.rawgit.com` to use a different CDN.
+
+Below is a video of the workflow:
+
+<div style="position:relative;height:0;padding-bottom:56.25%"><iframe src="https://www.youtube.com/embed/_D_C_oSKp9Y?ecver=2" width="640" height="360" frameborder="0" style="position:absolute;width:100%;height:100%;left:0" allowfullscreen></iframe></div>
+
+## Sharing Our Project
+
+Once we've published our awesome project, we'll want to share it so other
+people dive inside!
+
+### Creating Media
+
+[gifpardy]: https://github.com/ngokevin/gifpardy
+[obs]: https://obsproject.com/
+
+A-Frame and VR are very visual; we'll want to create videos and GIFs of our
+project.
+
+First, we want to record the screen. On OS X, record the screen using built-in
+QuickTime Player's Screen Recording or [OBS Studio][obs]. On Windows, we can
+use [OBS Studio][obs]. OBS Studio also supports streaming and compositing a
+webcam image on top of the screen, which is useful to show the person using the
+headset in reality (even with mixed reality).
+
+Then, we can possibly want to trim the video. On OS X, we can use QuickTime
+Player's trim tool (`<cmd> + t`).
+
+To convert to a GIF with one command, use [`gifpardy`]. `gifpardy` uses ffmpeg
+and gifsicle under the hood:
+
+```
+gifpardy in.mp4
+gifpardy in.mp4 out.gif
+gifpardy -r 320x240 --delay 8 in.mp4
+```
+
+Alternatively, we can capture straight to GIF using
+[LICECap](https://licecap.en.softonic.com/), but `gifpardy` is easy to produce
+smaller file sizes and sped-up GIFs.
```

Comment:
I'm very happy with [Gif Brewery](https://itunes.apple.com/us/app/gif-brewery-by-gfycat-capture-make-video-gifs/id1081413713?mt=12) — UI to trim, resize, crop, and preview your gif before exporting.